.github/workflows/ci.yaml

```diff
@@ -81,4 +95,39 @@ jobs:
       with:
         name: nvidiadriver-e2e-test-logs
         path: ./logs/
-        retention-days: 15  
+        retention-days: 15
+
+    - name: Precompiled e2e test- upgrade kernel and Validate gpu driver
+      env:
+        TEST_CASE_KERNEL_UPGRADE: "./tests/cases/nvidia-kernel-upgrade.sh"
```

Comment:
I wouldn't call this a test case, but rather, a helper script for upgrading the kernel. Can you move this script to the `scripts` folder and rename it to `upgrade-kernel.sh`?

Reply:
upgrade-kernel.sh renamed 
tests/precompile-task_executor.sh  added to handle multiple scripts file , reboot conditions